support/client/lib/vwf/model/jPlayer.js

```diff
@@ -342,6 +364,13 @@ define( [
 
     } );
 
+    function videoEndedCallback(){
+        var mediaManagerID = vwf.find( undefined, "/mediaManager" )[ 0 ];
+        var videoManagerID = vwf.find( mediaManagerID, "videoManager" ) [ 0 ];
+        console.log("Video ended callback in driver fired!");
+        vwf.fireEvent(videoManagerID, "videoEnded");
```

Comment:
A few issues here. We cannot assume an application will have any node named `mediaManager`, nor can we assume it will be a direct child of the scene. Same for `videoManager`. If these are nodes tracked by the driver, they need to be stored in `this.state.nodes`. The `console.log` statement needs to be removed. All direct uses of `vwf` need to be replaced. Now, finally, having said all of that, I'm curious as to why this callback exists here at all.